lib/elastic-search/requests.js

```diff
@@ -70,22 +70,11 @@ const _indexGeneric = async (indexName, records, update) => {
 
     const indexStatement = { _index: indexName, _id: uri }
 
-    if (process.env.ES_VERSION !== '8') {
-      indexStatement._type = 'resource'
-      // No longer need the _type (and it's going to throw an error bc it's redundant??):
-      delete record._type
-      if (record._parent) {
-        // No longer need the parent
-        indexStatement.parent = record._parent
-        delete record._parent
-      }
-    }
-
     const updatedAt = Date.now()
     record.updatedAt = updatedAt
 
     if (update) {
-      delete record.uri
+      // delete record.uri
```

Comment:
I'm not sure why we were doing this, but I'm pretty sure it's safe to remove? Otherwise the document processed stream write later on errors. Do we even use that stream?

Reply:
It's probably harmless. But I assume it was removed because updating a `uri` is redundant in the context of an update. This could probably be rearchitected to remove reliance on the `uri` property and allow us to continue to `delete record.uri`. (e.g. keep a log of bib uris updated and stop relying on those uris to exist in the poted documents.) But removing the `delete` call is probably fine for now.